### PR TITLE
adding in node tolerations to cluster-autoscaler deployment

### PR DESCRIPTION
**Pull request by ag237**
Ability to add node tolerations

### CODE REVIEW

File: stable/cluster-autoscaler/values.yaml [hunk 1]
```diff
@@ -14,6 +14,9 @@ image:
   tag: v0.6.0
   pullPolicy: IfNotPresent
 
+deployment:
```
Comment (unguiculus) [inline stable/cluster-autoscaler/values.yaml]: I don't think it makes sense to group it under `deployment`.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

**Review by mgoodness — CHANGES_REQUESTED** [review]
Don't forget to bump the chart version

**Comment by ag237** [comment]
I have made the suggested changes and bumped the chart version.

**Review by unguiculus — APPROVED** [review]

**Comment by unguiculus** [comment]
@mgoodness I guess we can ignore the failing test here.

**Review by mgoodness — APPROVED** [review]

**Comment by mgoodness** [comment]
> @mgoodness I guess we can ignore the failing test here.

Indeed.

/lgtm

**Pull request merged**